modules/apps/style-book/style-book-web-test/src/testIntegration/java/com/liferay/style/book/web/internal/portlet/action/test/ExportImportStyleBookEntriesMVCResourceCommandTest.java

```diff
@@ -117,12 +117,14 @@ public void testExportImportSingleStyleBookEntry() throws Exception {
 			ServiceContextTestUtil.getServiceContext(
 				_sourceGroup, TestPropsValues.getUserId());
 
+		String themeId = RandomTestUtil.randomString();
+
 		StyleBookEntry styleBookEntry =
 			_styleBookEntryLocalService.addStyleBookEntry(
 				null, TestPropsValues.getUserId(), _sourceGroup.getGroupId(),
 				false, _read("frontend-tokens-values.json"),
-				"Style Book Entry Name", "STYLE_BOOK_ENTRY_KEY",
-				RandomTestUtil.randomString(), serviceContext);
+				"Style Book Entry Name", "STYLE_BOOK_ENTRY_KEY", themeId,
```

Comment:
I know those were there when you arrived, but could you please update these tests to not use hardcoded strings unless they are necessary?

In other words, replace "Style Book Entry Name" (and similar ones) with `RandomTestUtil.randomString();`